codejail/safe_exec.py

```diff
@@ -238,13 +183,17 @@ def filter_unserializable(obj):
         elif isinstance(obj, list):
             new_list = []
             for i in obj:
-                try:
-                    new_obj = filter_unserializable(i)
-                    if jsonable(new_obj):
-                        new_list.append(new_obj)
-                except Exception:
-                    pass # Don't add the item if we can't decode it
+                new_obj = filter_unserializable(i)
+                if jsonable(new_obj):
+                    new_list.append(new_obj)
             return new_list
+        elif isinstance(obj, tuple):
+            list_for_new_tuple = []
```

Comment:
Since JSON serializes lists and tuples the same (as JSON arrays), you could treat them the same here too, and use the list code for both.